recipe/meta.yaml

```diff
@@ -37,7 +37,7 @@ test:
 
 about:
   home: https://github.com/benediktschmitt/py-filelock
-  license: Public Domain
+  license: Unlicense
```

Comment:
The license was updated to according to:

https://github.com/tox-dev/py-filelock/blob/main/LICENSE

https://spdx.org/licenses/Unlicense.html